include/auth_ejabberd.php

```diff
@@ -320,7 +324,7 @@ private function writeLog($sMessage) {
 	 * @param string $sMessage The logfile message
 	 */
 	private function writeDebugLog($sMessage) {
-		if ($this->bDebug)
+		if ($this->bDebug) {
```

Comment:
This bracket is never closed.

It would be really good if you could check your pull requests on your local installation for syntax errors.